tools/fmt/fmt.rs

```diff
@@ -192,9 +205,10 @@ fn format_component(
     state: &mut FormatState,
 ) -> Result<(), std::io::Error> {
     let mut sub = node.children_with_tokens();
-    let _ok = whitespace_to(&mut sub, SyntaxKind::DeclaredIdentifier, writer, state, "")?
-        && whitespace_to(&mut sub, SyntaxKind::ColonEqual, writer, state, " ")?
-        && whitespace_to(&mut sub, SyntaxKind::Element, writer, state, " ")?;
+
+    whitespace_to(&mut sub, SyntaxKind::DeclaredIdentifier, writer, state, "")?;
```

Comment:
removed these `let _ok` in multiple places - I believe they were never used

Reply:
the idea here is that  the `&&` operator make it stop if we did not find the particular token.

Maybe you can add a `SyntaxMatch::is_found()`  function or something so you can continue.

I had that to be conservative i think, but it might be important in case of syntax error, or if there are comments, so we don't format too much.

Reply:
oh that's a big brain short circuit.
now i'm noticing that my version indeed made the formatting worse with some of these comment/error scenarios. I'll roll this back!